site-src/api-types/grpcroute.md

```diff
@@ -0,0 +1,216 @@
+!!! info "Experimental Channel"
+
+    The `GRPCRoute` resource described below is currently only included in the
+    "Experimental" channel of Gateway API. For more information on release
+    channels, refer to the [related documentation](https://gateway-api.sigs.k8s.io/concepts/versioning).
```

Comment:
Tiny nit: Mind moving this below the GRPCRoute heading?

Reply:
Done.